src/graphql/typeDefs/dataTypes/milestone.graphql

```diff
@@ -1,6 +1,11 @@
+"THAT Milestone marker"
 type Milestone @key(fields: "id") {
+  "Unique milestone id"
   id: ID!
+  "Title of milestone"
   title: String!
+  "Description of this milestone"
   description: String
+  "The date associated with the milestone"
   dueDate: Date
```

Comment:
Shouldn't this be required?

Reply:
Good catch

Reply:
updated with commit cf4bfb72c7969d271fc7d2983b427d7a9a2de08a